src/ring_theory/ideal/quotient_operations.lean

```diff
@@ -126,6 +126,10 @@ lemma mem_quotient_iff_mem {I J : ideal R} (hIJ : I ≤ J) {x : R} :
   quotient.mk I x ∈ J.map (quotient.mk I) ↔ x ∈ J :=
 by rw [mem_quotient_iff_mem_sup, sup_eq_left.mpr hIJ]
 
+lemma comap_quo_map_quo {I J : ideal R} (h : I ≤ J) :
+  ideal.comap (ideal.quotient.mk I) (ideal.map (ideal.quotient.mk I) J) = J :=
+by { ext, rw [← ideal.mem_quotient_iff_mem h, ideal.mem_comap], }
```

Comment:
Can you prove this with `comap_map_of_surjective`?

Reply:
Here is what I could do
```lean
import ring_theory.ideal.quotient_operations

variables {R : Type*} [comm_ring R]

lemma comap_quot_map_quot {I J : ideal R} (h : I ≤ J) :
  ideal.comap (ideal.quotient.mk I) (ideal.map (ideal.quotient.mk I) J) = J :=
begin
  convert ideal.comap_map_of_surjective (ideal.quotient.mk I) (ideal.quotient.mk I).is_surjective _,
  rw (by convert ideal.mk_ker : ideal.comap (ideal.quotient.mk I) ⊥ = I),
  exact (sup_eq_left.mpr h).symm,
end
```

It is longer than the current proof. The result is a bit trivial so if you think it is not worthwhile, I can just prove it where I need it.

Reply:
I think the lemma is nice enough to have. Let's call it `comap_map_mk` to be consistent with `mk_ker` and `comap_map_of_surjective`.

Reply:
Ok. Naming lemma is indeed a difficult art ;)